pages/dao/[symbol]/proposal/components/instructions/UXDProtocol/EditController.tsx

```diff
@@ -0,0 +1,151 @@
+import * as yup from 'yup'
+
+import Switch from '@components/Switch'
+import { useContext, useEffect, useState } from 'react'
+import useGovernanceAssets from '@hooks/useGovernanceAssets'
+import {
+  Governance,
+  ProgramAccount,
+  serializeInstructionToBase64,
+} from '@solana/spl-governance'
+import useWalletStore from 'stores/useWalletStore'
+import { NewProposalContext } from '../../../new'
+import Input from '@components/inputs/Input'
+import {
+  UiInstruction,
+  UXDEditControllerForm,
+} from '@utils/uiTypes/proposalCreationTypes'
+import { isFormValid } from '@utils/formValidation'
+import { Controller, UXD_DECIMALS } from '@uxd-protocol/uxd-client'
+import { uxdClient } from '@tools/sdk/uxdProtocol/uxdClient'
+import GovernedAccountSelect from '../../GovernedAccountSelect'
+
+const schema = yup.object().shape({
+  governedAccount: yup
+    .object()
+    .nullable()
+    .required('Governance account is required'),
+  decimalRedeemableGlobalSupplyCap: yup
```

Comment:
What do you mean by `decimal` prefix in `decimalRedeemableGlobalSupplyCap`?

Reply:
ah they use that in Realms to say `uiAmount`. But here I just forgot to remove it (since we decided to only use prefixes for `native`

Reply:
fixed